### ISSUE TITLE
Plugin is not detected

### ISSUE THREAD

**Issue by s101d1** [samyBadjoudj/geany-qop-plugin#1]
I'm on Linux Mint 18 and I had followed the installation instruction in https://plugins.geany.org/quick_open_file.html (although I have to install libgtk2.0-dev first to successfully compile the source). Then I copied the qop.so to /usr/lib/x86_64-linux-gnu/geany folder. When I run the geany, the Plugin Manager doesn't show the Quick Open File plugin. There is no error message and file permission is fine.

**Comment by samyBadjoudj** [comment]
This folder is your Geany's libdir ?

**Comment by s101d1** [comment]
Yes I believe so, running `ls` there will show the built-in plugin .so files: classbuilder.so, export.so, filebrowser.so, etc. I also installed projectorganizer plugin and it has projectorganizer.so added there so I think the folder is correct.

**Comment by samyBadjoudj** [comment]
weird any log in verbose mode ?

**Comment by s101d1** [comment]
running `geany -v` gives:

Geany-INFO: Geany 1.27, en_US.UTF-8
Geany-INFO: GTK 2.24.30, GLib 2.48.1
Geany-INFO: System data dir: /usr/share/geany
Geany-INFO: User config dir: /home/user1/.config/geany
Geany-INFO: System plugin path: /usr/lib/x86_64-linux-gnu/geany
Geany-INFO: Added filetype Clojure (61).
Geany-INFO: Added filetype Cython (62).
Geany-INFO: Added filetype Scala (63).
Geany-INFO: Added filetype JSON (64).
Geany-INFO: Added filetype Genie (65).
Geany-INFO: Added filetype Graphviz (66).
Geany-INFO: Added filetype CUDA (67).
Geany-INFO: Loaded:   /usr/lib/x86_64-linux-gnu/geany/projectorganizer.so (Project Organizer)
Geany-INFO: unknown : None (UTF-8)

This is the output from `ls -l`:

-rw-r--r-- 1 root root 31280 Apr 18 04:46 classbuilder.so
-rw-r--r-- 1 root root 27264 Apr 18 04:46 export.so
-rw-r--r-- 1 root root 36256 Apr 18 04:46 filebrowser.so
-rw-r--r-- 1 root root 39680 Apr 18 04:46 htmlchars.so
-rw-r--r-- 1 root root 69896 Apr 18 15:19 projectorganizer.so
-rw-r--r-- 1 root root 19928 Sep  9 12:32 qop.so
-rw-r--r-- 1 root root 27616 Apr 18 04:46 saveactions.so
-rw-r--r-- 1 root root 19168 Apr 18 04:46 splitwindow.so

Could be the compiled qop.so is not compatible with geany? 
FYI I'm using linux mint 64 bit.

**Comment by samyBadjoudj** [comment]
hum  just compiled it and it is fine 
![qop](https://cloud.githubusercontent.com/assets/435516/18381488/c3daa2c0-767c-11e6-90f8-e91e70bed581.png)

try the following it is compiled for your arch
[qop.so.zip](https://github.com/samyBadjoudj/geany-qop-plugin/files/463594/qop.so.zip)

**Comment by s101d1** [comment]
Still same, but I found out something, if I opened Plugin Manager, the console showed this message:

Geany-INFO: Can't load plugin: /usr/lib/x86_64-linux-gnu/geany/qop.so: undefined symbol: document_grab_focus
Geany-INFO: Failed to load "/usr/lib/x86_64-linux-gnu/geany/qop.so" - ignoring plugin!

**Comment by samyBadjoudj** [comment]
Wich version of geany are u using ?

**Comment by s101d1** [comment]
1.27

**Comment by samyBadjoudj** [comment]
it seems that the api is not providing this method anymore.

https://github.com/geany/geany/issues/346

**Comment by s101d1** [comment]
Are you saying there is no fix for this?

**Comment by samyBadjoudj** [comment]
I guess there is a way to do it, by doing the same action but with the new api:

1. spot the removed functions 
2. look at the new api capabilities 
3. use the new api to do the same job if possible or implement the function

I think some functions might be in the same case...
